napari/layers/image/image.py

```diff
@@ -298,7 +298,7 @@ def __init__(
         self.interpolation = interpolation
         self.rendering = rendering
         if plane is not None:
-            self.plane.update(plane)
+            self.plane = plane
```

Comment:
should the `None` check happen in the setter?

Reply:
and if it's None, set to disabled, or something like that.

Reply:
I left it this way because I guess the setter is called more than the init but I don't feel strongly, happy to change if you prefer

Reply:
But what happens if I do `layer.plane = None`? I think that's likely to happen if someone is trying to disable it... not sure :) This was more obvious for clipping planes!

Reply:
Okay I've changed so that the setter is used and the default argument is a `PlaneManager()` instance, I don't think interpreting setting to `None` as disabling is correct here, it will fail with an error in this case

Reply:
Sounds good!